config/configuration_keys.json

```diff
@@ -1,8 +1,13 @@
 
 {
-    "awsAccessKeyId": "XXXX", 
-    "awsSecretAccessKey":"XXXX",  
-	"region" : "XXXX", 
-	"usersbucket": "XXXX" 
+    "awsAccessKeyId": "XXXXXXXXXXXXX",
```

Comment:
Please change this to configuration_keys.example.json

Reply:
Renamed configuration_keys.json to configuration_keys.example.json . Please review new changes.